lib/util/parseJsonFile.js

```diff
@@ -1,13 +1,19 @@
-const { existsSync, readFileSync } = require("fs");
-const { format } = require("util");
+const { readFile, stat } = require("fs");
+const { promisify } = require("util");
 
+const asyncStat = promisify(stat);
+const asyncReadFile = promisify(readFile);
 
 module.exports = {
-    parseJsonFile: path => {
-        if (!existsSync(path)) {
-            Error(format('%s it\' not a file', path));
+    parseJsonFile: async path => {
+        const stats = await asyncStat(path);
+
+        if (!stats.isFile()) {
+            throw `'${path}' is not a file`;
         }
+                    
+        const fileRead = await asyncReadFile(path);
 
-        return JSON.parse(readFileSync(path).toString())
+        return JSON.parse(fileRead.toString());        
```

Comment:
```suggestion
        return JSON.parse(fileRead.toString());
```

Reply:
I use this [extension](https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces), to help avoid blank